docs/flink-queries.md

```diff
@@ -80,6 +80,21 @@ Here are the SQL settings for the [FLIP-27](https://cwiki.apache.org/confluence/
 SET table.exec.iceberg.use-flip27-source = true;
 ```
 
+### Branch and Tag Read
```

Comment:
Do we add them to SQL and DataStream, respectively?

Reply:
Sorry not sure I follow? do you mean have branching and tagging be a subsection of SQL and Data Stream? If so I was preferring to have a separate section but I'm not super opinionated on that, lmk what you think!

Reply:
I think he is talking about adding examples to `Reading with DataStream`

Reply:
Sure, I've added examples!